src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -364,6 +364,9 @@ void InitCapabilities() {
                      hmi_apis::Common_ImageFieldName::locationImage));
```

Comment:
`subtleAlertText1`, `subtleAlertText2` and `subtleAlertSoftButtonText` should be inserted into `text_fields_enum_name`